spark/spark-spa/src/main/java/com/thoughtworks/go/spark/spa/PipelinesController.java

```diff
@@ -45,7 +45,7 @@ public String controllerBasePath() {
     @Override
     public void setupRoutes() {
         path(controllerBasePath(), () -> {
-            before("", authenticationHelper::checkAdminUserAnd403);
+            before("", authenticationHelper::checkAdminUserOrGroupAdminUserAnd403);
```

Comment:
No tests for this?

Reply:
Looks like the tests are there now in `spark/spark-spa/src/test/groovy/com/thoughtworks/go/spark/spa/PipelinesControllerTest.groovy`:

```groovy
    @Nested
    class Security implements SecurityTestTrait, GroupAdminUserSecurity { ... }
```